mmv1/templates/terraform/examples/gkehub_membership_binding_basic.tf.erb

```diff
@@ -15,22 +15,22 @@ resource "google_gke_hub_membership" "<%= ctx[:primary_resource_id] %>" {
   depends_on = [google_container_cluster.primary]
 }
 
-resource "google_gke_hub_scope" "<%= ctx[:primary_resource_id] %>" {
+resource "google_gke_hub_scope" "scope" {
   scope_id = "tf-test-scope%{random_suffix}"
 }
 
-resource "google_gke_hub_membership_binding" "<%= ctx[:primary_resource_id] %>" {
+resource "google_gke_hub_membership_binding" "membership_binding" {
```

Comment:
```suggestion
resource "google_gke_hub_membership_binding" "<%= ctx[:primary_resource_id] %>" {
```
Let's keep it for `google_gke_hub_membership_binding` since this is the primary resource for the test, and we may use it for some other sample/ test generation that has the dependency on primary_resource_id. I see that you've changed the name in the yaml file and I believe the name should be change to `membership_binding` already.